WooCommerce/Classes/ViewRelated/Products/Variations/Add Attributes/AddAttributeOptionsViewModel.swift

```diff
@@ -285,7 +297,7 @@ extension AddAttributeOptionsViewModel {
         // Name has to be considered, because local attributes are zero-id based.
         let updatedAttributes: [ProductAttribute] = {
             var attributes = product.attributes
-            attributes.removeAll { $0.attributeID == newAttribute.attributeID && $0.name == newAttribute.name }
+            attributes.removeAll { $0.attributeID == attribute.attributeID && $0.name == attribute.name }
```

Comment:
Tricky! It's probably worth it to add a comment explaining that we use `attribute` and not `newAttribute` to make sure the old name is the one that is matched, because the new name could not match!

Reply:
Comment added in https://github.com/woocommerce/woocommerce-ios/pull/3781/commits/d8f7de5cafd75a2e4c06ca34a267ed2b08d757bb.